ci/backend_recipes/omniscidb/meta.yaml

```diff
@@ -0,0 +1,62 @@
+# This is a copy of https://github.com/conda-forge/ibis-framework-feedstock/blob/master/recipe/meta.yaml
+# Changes required to the recipe will be performed and tested with this file during
+# development of Ibis, and this file needs to replace the original one on releases.
+#
+# Changes to the original file that need to be restored when copying to release:
+# - Set the version in the first line: {% set version = "1.3.0" %}
+# - Add `sha256` key to the `source` section, with the tar.gz hash
+# - Set the `number` in the `build` section to the appropriate build number
+# - Remove this comment from the beginning of the file
+
+package:
+  name: ibis-framework[omniscidb]
+  version: {{ version }}
+
+source:
+  url: https://github.com/ibis-project/ibis/archive/{{ version }}.tar.gz
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  skip: true  # [py<37]
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+
+  run:
+    - geopandas
+    - numpy >=1.15
+    - pandas >=0.25.3
+    - pyarrow >=0.15
+    - pymapd 0.24  # [py<38]
```

Comment:
likley you can make this mostly about pymapd, it will pull in whatever else is needed. (e.g. don't this the directs unless you need to restrict them for some reason)

this should depend on ibis-core package